packages/myst-cli/src/process/mdast.ts

```diff
@@ -230,6 +232,8 @@ export async function transformMdast(
     .run(mdast, vfile);
   transformImagesToEmbed(mdast);
   transformImagesWithoutExt(session, mdast, file, { imageExtensions });
+  const isJupytext = frontmatter.kernelspec || frontmatter.jupytext;
+  if (isJupytext) transformLiftCodeBlocksInJupytext(mdast);
```

Comment:
I think this is ok running here - The other block/code transformations happen pretty early, but I don't think there is anything in here that is order-dependent.